Core/src/org/sleuthkit/autopsy/corecomponents/Bundle.properties

```diff
@@ -201,3 +200,5 @@ DataResultViewerThumbnail.sortButton.text=Sort
 CriterionChooser.ascendingRadio.text=\u25b2 Ascending\n
 CriterionChooser.removeButton.text=Remove
 CriterionChooser.descendingRadio.text=\u25bc Descending
+AutopsyOptionsPanel.defaultLogoRB.text=Use default
+AutopsyOptionsPanel.specifyLogoRB.text=Specify a logo
```

Comment:
New messages should be added as @Messages instead of directly to the Bundle.properties files